Sources/PackageModel/SwiftSDKs/SwiftSDKConfigurationStore.swift

```diff
@@ -141,4 +157,92 @@ public final class SwiftSDKConfigurationStore {
         try fileSystem.removeFileTree(configurationPath)
         return true
     }
+
+    public func configure(
+        sdkID: String,
+        targetTriple: String?,
+        showConfiguration: Bool,
+        resetConfiguration: Bool,
+        config: SwiftSDK.PathsConfiguration<String>
+    ) throws -> Bool {
+        let targetTriples: [Triple]
+        if let targetTriple = targetTriple {
+            targetTriples = try [Triple(targetTriple)]
+        } else {
+            // when target-triple is unspecified, configure every triple for the SDK
```

Comment:
comment formatting nit
```suggestion
            // when `targetTriple` is unspecified, configure every triple for the SDK
```